libs/shared/contentful/src/lib/contentful.stripe-mapper.spec.ts

```diff
@@ -0,0 +1,337 @@
+import { Stripe } from 'stripe';
+import { ContentfulClient } from './contentful.client';
+import { ContentfulManager } from './contentful.manager';
+import { ContentfulStripeMapper, PlanMapper } from './contentful.stripe-mapper';
+import {
+  OfferingCommonContentResult,
+  PurchaseDetailsTransformed,
+  PurchaseWithDetailsOfferingContentTransformed,
+} from './queries/purchase-with-details-offering-content';
+import {
+  OfferingCommonContentResultFactory,
+  PurchaseDetailsTransformedFactory,
+  PurchaseWithDetailsOfferingContentTransformedFactory,
+} from './queries/purchase-with-details-offering-content/factories';
+import { ProductFactory, PlanFactory } from '@fxa/payments/stripe';
+import { StripeMetadata, StripeMetadataKeys } from './types';
+import { MetadataFactory } from './factories';
+
+jest.mock('./contentful.manager');
+
+describe('PlanMapper', () => {
+  const defaultCommonContent: OfferingCommonContentResult =
+    OfferingCommonContentResultFactory();
+  const defaultPurchaseDetails: PurchaseDetailsTransformed =
+    PurchaseDetailsTransformedFactory();
+  const stripeMetadata: StripeMetadata = MetadataFactory({
```

Comment:
These shouldn't need the typecasts since their type comes directly at assignment from the factories